src/main/java/com/playmonumenta/libraryofsouls/bestiary/BestiaryArea.java

```diff
@@ -48,6 +48,13 @@ public BestiaryArea(@Nullable BestiaryArea parent, String name, ConfigurationSec
 			if (souls == null || souls.isEmpty()) {
 				throw new Exception("Bestiary entry " + Utils.plainText(mName) + " specifies nonexistent location " + mLocation);
 			}
+			//Truly awful workaround. Big fan of ampersands
+			if (config.contains("ignored_souls")) {
+				String[] noSouls = config.getString("ignored_souls").split("&");
+				for (String s : noSouls) {
+					souls.remove(SoulsDatabase.getInstance().getSoul(s));
+				}
+			}
```

Comment:
Why would we do this via goofy string splitting? This config element can certainly be a normal list...?